internal/pkg/cli/local_run.go

```diff
@@ -314,6 +330,10 @@ func (o *localRunOpts) runPauseContainer(ctx context.Context, containerPorts map
 		ContainerName:  containerNameWithSuffix,
 		ContainerPorts: containerPorts,
 		Command:        []string{"sleep", "infinity"},
+		LogOptions: dockerengine.RunLogOptions{
+			Color:      o.newColor(),
+			LinePrefix: "[pause] ",
```

Comment:
Non-blocking: I sometimes wonder if we should name `pause` something else, although I know that's how ECS names it 😂